spec/cable/connection_spec.cr

```diff
@@ -24,16 +24,13 @@ describe Cable::Connection do
 
     it "accepts without params hash key" do
       connect do |connection, socket|
-        connection.receive({"command" => "subscribe", "identifier" => {channel: "ChatChannel", room: "1"}.to_json}.to_json)
```

Comment:
took me a while to figure out about this change, then I read the test description 🤦‍♂️ lol